### PR TITLE
Change `vt` to `view-transition` and add a duration

### PR DESCRIPTION
**Pull request by bencroker**
This PR changes the fragment event option `vt` to `view-transition` and turns it into it a duration. If set to `0`, view transitions are not applied, otherwise they are using the provided duration.

### CODE REVIEW

File: packages/library/src/lib/plugins/backend.ts [hunk 1]
```diff
@@ -439,7 +439,7 @@ export function mergeHTMLFragment(
 
         const revisedHTML = modifiedTarget.outerHTML
 
-        if (originalHTML !== revisedHTML) {
+        if (originalHTML !== revisedHTML && settleTime > 0) {
```
Comment (bencroker) [inline packages/library/src/lib/plugins/backend.ts]: @delaneyj I added a condition to only add/remove the settling class if the settle time is non-zero. Please review if you’re ok with this.

**Comment by delaneyj** [comment]
need all examples and helpers updated too!

**Comment by bencroker** [comment]
This is ready for review. Please carefully check my work in [backend.ts](https://github.com/delaneyj/datastar/pull/167/files#diff-7afc62c866ea4a0c5ba0c1f11c8d901fae4f1193ff4339fa0bece49f85e822db).

**Comment by bencroker** [comment]
Since all that’s required for View Transition animations is CSS, I no longer think animation duration is something Datastar should handle. We’d only be allowing setting the duration, not any of the other animation effects, so my feeling is that it’s probably better to just allow enabling/disabling View Transition and leave the rest to the user, as is the case now. 

I’d still like to see `vt` renamed to `view-transition`, but understand that it is a breaking change in the name of ergonomics.

**Pull request closed without merging**